src/components/sbb-no-results/sbb-no-results.scss

```diff
@@ -0,0 +1,45 @@
+@use '../../global/styles' as sbb;
+
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include sbb.host-component-properties;
+
+:host {
+  --sbb-no-results-subtitle-color: var(--sbb-color-granite-default);
+  --sbb-no-results-image-margin-block: 0 var(--sbb-spacing-responsive-s);
+  --sbb-no-results-legend-margin-block: var(--sbb-spacing-responsive-xxxs) 0;
+  --sbb-no-results-action-margin-block: var(--sbb-spacing-responsive-xxxs) 0;
+
+  display: flex;
```

Comment:
Is the `display: flex` really needed? The `text-align: center` might be sufficient (?). And would it be possible to move the styles from the host to an inner container for consistency/stability? What do you think?

```scss
 .sbb-no-results__container {
  text-align: center;
}
```